### DISCUSSION TITLE
Recipe image not loading

### DISCUSSION THREAD

**Discussion by FiresideTale** [TandoorRecipes/recipes#1029] [General]
Hi, might be something simple I'm missing, but since updating to version 1.0.0 none of the images are loading for my recipes. I'm on Synology and can still see the images in the mediafiles folder, but on my Tandoor they're failing to load. All local network fyi. 

Not 100% sure if it's an issue of my own, so that's why I'm not reporting as a bug.  

Any help would be appreciated. Thanks :)

**Comment by vabene1111** [comment]
from which verson did you upgrade to 1.0 ? we have not really changed anything related to image serving in quite a while but if you were on a really old versions i might be able to point you to a few places that could be the reason of images not loading. Also dont worry about creationg issues, its one for you so we can solve it together :)
